web/ui/react-app/src/pages/graph/SeriesName.tsx

```diff
@@ -16,7 +34,7 @@ const SeriesName: FC<SeriesNameProps> = ({ labels, format }) => {
       }
 
       labelNodes.push(
-        <span key={label}>
+        <span className="legend-label-container" onClick={toClipboard} key={label} title="Click to copy label matcher">
```

Comment:
I noticed that the hovering now includes the comma between labels as well.

How about changing the HTML here so that the `legend-label-container` only wraps the k=v part without the comma, like:

```tsx
        <span key={label}>
          {!first && ', '}
          <span className="legend-label-container" onClick={toClipboard} title="Click to copy label matcher">
            <span className="legend-label-name">{label}</span>=<span className="legend-label-value">"{labels[label]}"</span>
          </span>
        </span>
```

Then you can also drop the logic around the comma in `toClipboard`.

Reply:
Thought of that too. I have to confess that i did not go with that approach since it would require to change the existing  tests a little bit 😄... but it is the proper way to go.